util/make_timeline.py

```diff
@@ -134,9 +188,62 @@ def parse_file(args):
 
     return entries, last_ability_time
 
+def find_fights_in_file(file):
+    e_starts = []
+    encounter_start_staging = 0 # Staged to avoid spurious starts such as limit break usage
+    e_ends = []
+    encounter_in_progress = False
+    current_instance = None
+    encounter_sets = []
+    for line in file:
+        # Ignore log lines that aren't game status info
+        if not (line[37:41] == '0839' or line[0:2] in ['00', '01', '21', '22', '33']):
```

Comment:
I think [De Morgan's laws](https://en.wikipedia.org/wiki/De_Morgan%27s_laws) finally come in handy!
I think
```python3
if line[37:41] != '0839' and line[0:2] not in ['00', '01', '21', '22', '33']:
```
should resolve slightly faster

Reply:
That's a good point, I hadn't considered short-circuit evaluation. It's enough faster to be noticeable, thanks for pointing that out!